cmd/skupper/skupper_kube_expose.go

```diff
@@ -149,3 +149,16 @@ func (s *SkupperKubeService) UnexposeFlags(cmd *cobra.Command) error {
 	cmd.Use = "unexpose [deployment <name>|pods <selector>|statefulset <statefulsetname>|service <name>|deploymentconfig <name>]"
 	return nil
 }
+
+func (s *SkupperKubeService) UnExposeArgs(cmd *cobra.Command, args []string) error {
+	if len(args) < 1 || (!strings.Contains(args[0], "/") && len(args) < 2) {
+		return fmt.Errorf("target and name must be specified (e.g. 'skupper unexpose deployment <name>')")
```

Comment:
Maybe create an unexported method used by both `ExposeArgs` and `UnexposeArgs` that
simply gets the command name to compose the error message using `cmd.Name()`.

Reply:
updated in e9cd7496aa1548fcb4cbb73325afa2ab6347badd